Detectors/MUON/MCH/Workflow/src/ClusterSamplerSpec.cxx

```diff
@@ -66,40 +75,77 @@ class ClusterSamplerTask
   //_________________________________________________________________________________________________
   void run(framework::ProcessingContext& pc)
   {
-    /// send the clusters of the current event
+    /// send the clusters of the next events in the current TF
+
+    static uint32_t event(-1);
```

Comment:
unsigned int initialised to negative value ?

Reply:
I prefer to fix it now because I don't have plan to re-touch this code soon.